canary/canary_test.py

```diff
@@ -0,0 +1,75 @@
+import asyncio
+import datetime
+import os
+import time
+
+from pymongo import MongoClient
+from typing import Coroutine, Any
+from viam.robot.client import RobotClient
+from viam.components.board import Board
+
+async def connect(robot_address: str, api_key: str, api_key_id: str) -> Coroutine[Any, Any, RobotClient]:
+    opts = RobotClient.Options.with_api_key(
```

Comment:
There were some recommended connection settings for python clients identified in https://viam.atlassian.net/browse/RSDK-6144. Not sure if those are 1) still relevant after @npmenard's recent improvements, or 2) something we want to do here if we are trying to measure "default" quality.